server/controllers/azuread.js

```diff
@@ -73,12 +74,20 @@ async function azureAdSignInCallback(ctx) {
         "Content-Type": "application/x-www-form-urlencoded",
       },
     });
-    const userResponse = await axios.get(OAUTH_USER_INFO_ENDPOINT, {
+    const apiResponse = await axios.get(isOIDC?OAUTH_USER_INFO_ENDPOINT:'https://graph.microsoft.com/v1.0/me', {
       headers: {
         Authorization: `Bearer ${response.data.access_token}`,
       },
     });
 
+    const userResponse = isOIDC ? apiResponse : {
+      data: {
+        email: apiResponse.data.email,
```

Comment:
@luokui-c12 
Sorry for the delay in checking.
This is an `email`, but is it not a `mail` correctly?